File: scala-gatling-client/src/gatling/scala/io/swagger/client/model/UserTokenObject.scala

package io.swagger.client.model


case class UserTokenObject (
    _id: String,
    _username: String,
    __meta: _metaObject,
    _sessiontoken: String
)
object UserTokenObject {
    def toStringBody(var_id: Object, var_username: Object, var__meta: Object, var_sessiontoken: Object) =
        s"""
        | {
        | "id":$var_id,"username":$var_username,"_meta":$var__meta,"sessiontoken":$var_sessiontoken
        | }
        """.stripMargin
}

File: dart-jaguar-client/lib/model/user_token_object.dart
import 'package:jaguar_serializer/jaguar_serializer.dart';

import 'package:swagger/model/meta_object.dart';
part 'user_token_object.jser.dart';

class UserTokenObject {
  
  final String id;
  
  final String username;
  
  final MetaObject meta;
  
  final String sessiontoken;
  

  UserTokenObject(
    

{
    
     this.id = null,  
     this.username = null,  
     this.meta = null,  
     this.sessiontoken = null 
    }
  );

  @override
  String toString() {
    return 'UserTokenObject[id=$id, username=$username, meta=$meta, sessiontoken=$sessiontoken, ]';
  }
}

@GenSerializer()
class UserTokenObjectSerializer extends Serializer<UserTokenObject> with _$UserTokenObjectSerializer {

}
